src/components/Controls/ArrayInput/index.tsx

```diff
@@ -0,0 +1,56 @@
+import {FunctionComponent} from "react"
+import CloseIcon from "../../../assets/icons/CloseIcon"
+import Input from "../Input"
+import Button from "../Button"
+import useArrayInput from "./hooks"
+import "./styles.scss"
+
+export type ArrayInputChangeListener = (newValue: ArrayInputValue) => void
+export type ArrayInputValue = string[]
+
+const ArrayInput: FunctionComponent<{
+	onChange: ArrayInputChangeListener
+	onRemove: (indexToRemove: number) => void
+	value: string[]
+	validation?: string | null
+	borders?: "all" | "bottom"
+	placeholder?: string
+}> = ({
+	onChange,
+	onRemove,
+	value,
+	borders,
+	validation,
+	placeholder = "Type value and press enter or tab..."
+}) => {
+	const {handleInputChange, handleKeyDown, inputValue} = useArrayInput({
+		onChange,
+		value
+	})
+	return (
+		<section className="array-input-container">
```

Comment:
First, please use BEM naming methodology for classes.
Second, why it is a section? Section is intended to be a large section of the page.

Reply:
Upd, I see you are using BEM, but why such a long name? Why not just array-input?